pluto_build/sql/update_empty_coord.sql

```diff
@@ -0,0 +1,34 @@
+-- Update X/Y and Lat/Long fields that are NULL with values from Lot Centroids
+-- Make sure lot centroids fall within the polygon
+SELECT bbl, centroid, point_surface,
```

Comment:
This code makes sense to me and is readable. I would use a temporary table for `update_table` instead of creating and dropping. https://www.postgresqltutorial.com/postgresql-tutorial/postgresql-temporary-table/

Instead of storing the centroid and "point on surface" as two different columns, you could use a `CASE` statement to write them to the same column. Then you would only need one update query.

Reply:
second Sasha's point about creating temp table which could also use the table comprehension syntax which is common syntax in our data product. Namely `WITH update_data as` many examples throughout pluto as well e.g. [here](https://github.com/NYCPlanning/db-pluto/blob/d433ed94b1ea8d7c636cb6ddf296bdf7140de7d9/pluto_build/sql/cama_easements.sql#L3-L16)

Reply:
Cool. Will modify it.